scenes/help.py

```diff
@@ -5,34 +5,27 @@
 from bearlibterminal import bltutils
 import common.utils as utils
 
-y = 12
-x = 2
+line = 12
+offset = 10
 
 def keybindings():
 
-    i = x
-    j = y
+    i = offset
```

Comment:
some names are good, but not all, for example, what are i and j stand for ? hard to guess without looking through the code, and what is h for, basically it is not short for height, because width is used, then what ?

Reply:
Fixed.